controllers/olympic.js

```diff
@@ -0,0 +1,46 @@
+const Olympic = require("../models/Olympic");
+const { StatusCodes } = require("http-status-codes");
+const { BadRequestError, UnauthenticatedError } = require("../errors");

```

Comment:
These imports are not being used in this file.

Reply:
I utilize  { StatusCodes } now boy invoke  status(StatusCodes.OK).send.... instead of status(200).send...
I belive we don't need UnauthenticatedError here however do we need BadRequestError handle ?